src/components/WeekDay/index.js

```diff
@@ -0,0 +1,116 @@
+import React, {useState} from 'react';
+import {
+  Container,
+  Week,
+  Day,
+  WeekDayContainer,
+  DayContainer,
+  TemperatureContainer,
+  MaxTemperature,
+  MinTemperature,
+  RainContainer,
+  Title,
+  Description,
+  HumidtyContainer,
+  ConteinerIcon,
+  ContainerDescription,
+} from './styles';
+import Icon from 'react-native-vector-icons/MaterialCommunityIcons';
+
+export default function WeekDay() {
+  const [dados] = useState([
+    {
+      id: 1,
+      icon: 'weather-cloudy',
+      day: 'Monday',
+      maxTemperature: '30º',
```

Comment:
Não sei se é interessante para o projeto hoje mas pode ser um TODO para o futuro: A opção de fazer um switch de Celsius para Fahrenheit. Existem alguns países que ainda o utilizam (https://pt.ripleybelieves.com/countries-that-use-fahrenheit-2220)